agent/tool-scripts/pcp-transient

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+# -*- mode: python -*-
+
+import sys
+import os
+
+PROG = os.path.basename(sys.argv[0])
+
+if len(sys.argv) != 2 or sys.argv[1] != "--help":
+    print(f"{PROG}: This script is deprecated, please run it with --help for info on registering the tool.", file=sys.stderr)
+    print(f"{PROG}: Run {os.path.abspath(__file__)} --help for more info.", file=sys.stderr)
+    sys.exit(1)
+
+if sys.argv[1] == "--help":
+    print(f"{PROG}: Options:", file=sys.stdout)
+    print(f"{PROG}: No tool-specific options currently", file=sys.stdout)
```

Comment:
This is unfortunately a relic of the old tool system. Tools used to have bash scripts in this folder that contained instructions for their start, stop, postprocess, etc. Now that is all encapsulated within the tool meister (for new tools), but the tool registration system uses the existence of those files to check whether a tool can be registered. Additionally, it will always recommend you run `<path>/tool-scripts/<tool_name> --help` to see all of the tool-specific options. These confusing messages actually mirror the functionality (and exact text output) of almost every other existing tool currently, so I'll open up a new issue to fix this poorly-aging system.

Reply:
Addressed in: https://github.com/distributed-system-analysis/pbench/issues/2204